packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -95,12 +103,35 @@ function sortByTimestampDescending(a: Transfer, b: Transfer) {
 }
 
 function getMultiChainFetchList(): ChainPair[] {
+  const allChains = getChains()
+
   return getChains().flatMap(chain => {
     // We only grab child chains because we don't want duplicates and we need the parent chain
     // Although the type is correct here we default to an empty array for custom networks backwards compatibility
-    const childChainIds = chain.partnerChainIDs ?? []
+    let childChainIds = chain.partnerChainIDs ?? []
```

Comment:
We may not be able to avoid having `let childChainIds`, but I think `addTeleportDestinationChainToList` is a bit overengineered, we could do:
```
let childChainIds = chain.partnerChainIDs ?? []
childChainIds.push(...TELEPORT_ALLOWLIST[chainId])

// dedupe
childChainIds = Array.from(new Set(childChainIds))
```